images_config/ks-robot-master.cfg

```diff
@@ -0,0 +1,456 @@
+##############NewUser#################
+############FOR ROBOT-LOCAL USER##########################
+sed -i 's/HOME=/home/HOME=/iscsi/user/g' /etc/default/useradd
+
+adduser --disabled-password --gecos "" robot-local --home /home/robot-local
+
+echo "robot-local:robot" | chpasswd
+echo "robot-local ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+usermod -m -d /home/robot-local robot-local
+mkdir /home/robot-local
+chown robot-local:robot-local /home/robot-local
+ssh-add
+##################ROBOT USER###########################
+sed -i 's/HOME=/home/HOME=/iscsi/user/g' /etc/default/useradd
+
+adduser --disabled-password --gecos "" robot --home /u/robot
+echo 'robot:robot' | chpasswd
+
+echo "robot ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+usermod -m -d /u/robot robot
+mkdir /u/robot
+cp -rT /etc/skel /u/robot
+chown robot:robot /u/robot
+ssh-add
+##############################################################
+
+
+
+%end
+
+%post
+#########################################
+echo "\n${green}INFO:Setup local ROOT user${NC}\n"
+  sleep 5
+#passwd d root
+#/usr/bin/passwd -d root
+
+echo "Defaults rootpw" >> /etc/sudoers
+rootpw robot
+#passwd root --iscrypted $1$tEHboVqc$8dkJ2OvCfg8P3z/ru2puI0
+
+
+echo "robot-local ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
+echo "%users ALL=NOPASSWD:/usr/sbin/cob-shutdown" >> /etc/sudoers
+###echo " Defaults env_keep += "https_proxy" " >> /etc/sudoers
+sed '$ a Defaults env_keep += "https_proxy"' /etc/sudoers
+
+
+wget repository
+
+apt-get -y -q update
+
+sleep 20
+
+%end
+
+%post --interpreter /bin/bash
+##echo "Defaults env_keep += "https_proxy"" >> /etc/sudoers
+
+echo -e "\n${green}INFO:  Install NFS${NC}\n"
+sleep 5
+apt-get install lightdm -y
+
+dpkg-reconfigure lightdm
+apt-get install nfs-kernel-server autofs -y -f -y
+mkdir /u
+
+##grep -q -F '/home /u none bind 0 0' /etc/fstab || sh -c '(echo "/home /u none bind 0 0") > /etc/fstab'
+##echo "/home /u none bind 0 0" >> /etc/fstab
+##mount /u
+
+grep -q -F '/u *(rw,fsid=0,sync,no_subtree_check)' /etc/exports || sh -c '(echo "/u *(rw,fsid=0,sync,no_subtree_check)") >> /etc/exports'
+echo "/u *(rw,fsid=0,sync,no_subtree_check)" >> /etc/exports
+sed -i 's/NEED_STADT\=/NEED_STADT\=yes/g' /etc/default/nfs-common
+if [ -d "/u/robot/git" ]
+then
+sed -i 's/home\/robot/u\/robot/g' /etc/passwd
+read -p "Reboot the computer for the changes to take effect (y/n)?" choice
+case "$choice" in 
+y|Y ) echo "Yes" && reboot;;
+n|N ) echo "No" && exit;;
+* ) echo "invalid"&& exit;;
+esac
+fi
+#############ssh-gedit-display ########################
+
+echo "X11Forwarding yes" >> /etc/ssh/sshd_config
+
+echo "X11UseLocalhost no" >> /etc/ssh/sshd_config
+echo "PermitRootLogin yes">> /etc/ssh/sshd_config
+/etc/init.d/ssh restart
+#######################################################
+##########################NTP CONFIGURATION############################
+HOST=$(cat /etc/hostname)
+echo $HOST
+
+echo "# Specify one or more NTP servers." >> /etc/ntp.conf
```

Comment:
Please use wget also here:

`wget -O /etc/ntp.conf https://raw.githubusercontent.com/ipa320/setup_cob4/master/cob-pcs/ntp_server`

and with sed write the servername